src/Symfony/Component/Form/Tests/Extension/Core/Type/FileTypeTest.php

```diff
@@ -203,7 +203,7 @@ public function testFailedFileUploadIsTurnedIntoFormErrorUsingHttpFoundationRequ
             ->createBuilder(static::TESTED_TYPE)
             ->setRequestHandler(new HttpFoundationRequestHandler())
             ->getForm();
-        $form->submit(new UploadedFile(__DIR__.'/../../../Fixtures/foo', 'foo', null, null, $errorCode, true));
+        $form->submit(new UploadedFile(__DIR__.'/../../../Fixtures/foo', 'foo', null, $errorCode, true));
```

Comment:
adjustment was needed after merging #30895 up into the `4.2` branch as the filesize argument was deprecated in the meantime